src/network/serverpackethandler.cpp

```diff
@@ -629,13 +629,19 @@ void Server::handleCommand_InventoryAction(NetworkPacket* pkt)
 		if (ma->from_inv != ma->to_inv)
 			m_inventory_mgr->setInventoryModified(ma->to_inv);
 
-		bool from_inv_is_current_player =
-			(ma->from_inv.type == InventoryLocation::PLAYER) &&
-			(ma->from_inv.name == player->getName());
-
-		bool to_inv_is_current_player =
-			(ma->to_inv.type == InventoryLocation::PLAYER) &&
-			(ma->to_inv.name == player->getName());
+		bool from_inv_is_current_player = ma->from_inv.type != InventoryLocation::CURRENT_PLAYER;
```

Comment:
This is still incorrect
the action is from current players inventory if `ma->from_inv.type == InventoryLocation::CURRENT_PLAYER`

Reply:
Sorry, should be fixed now